### ISSUE TITLE
WPS: CDFile object endless destructor recursion

### ISSUE THREAD

**Issue by pacesm** [EOxServer/eoxserver#330]
There is a bug in the CDFile object which causes endless recursion in the CDFile object destructor. Can be checked by:
```
from eoxserver.services.ows.wps.parameters.complexdata import CDFile
obj = CDFile("test")
del obj
```

The bug is caused by missing _file attribute required by the destructor. The attribute is not created because of an error in the object constructor.

The fix is straightforward and it will be provided quickly.

**Comment by pacesm** [comment]
solution in 0.4 branch provided by PR  #331

**Comment by pacesm** [comment]
Fixed in master by the latest 0.4 branch merge.
Issue can be closed.
